src/controllers/appointment.ts

```diff
@@ -0,0 +1,23 @@
+import { NextFunction, Request, Response } from 'express';
+import getAppointmentsPerDateService from '../services/appointment';
+import { templateErrors } from '../helpers';
+
+const getAppointments = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  try {
+    const date = req.query.date as string;

```

Comment:
I think we need to validate the date parameter and handle unexpected formats or missing values.

Reply:
I do agree

Reply:
done